### PR TITLE
Add initial version of compatiblity scenario text objects

### PR DESCRIPTION
**Pull request by AaronVanGeffen**
Dumped using a script: [readParkStrings.py](https://gist.github.com/AaronVanGeffen/fc3570db40c1839e26ccc977b7209dc8)

### CODE REVIEW

**Review by tupaschoal — COMMENTED** [review]
I love this idea of finally getting rid of that weird piece of text on the strings file.

Not sure how we can help to review, it's certainly too much to manually review. I don't love the indentation used, but then the automatic JSON formatters are usually pretty bad, so no big deal.

**Comment by Gymnasiast** [comment]
ZehMatt has written one specifically for this repo. I can run it on the files once they have been generated.

File: objects/official/scenario_text/official.scenario_text.beneath_the_christmas_tree.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+   "eo-ZZ": "Sub la Kristnaska Arbo",
+   "es-ES": "Bajo el Árbol de Navidad",
+   "fi-FI": "Alla joulupuun",
+   "fr-FR": "Sous le sapin de Noël",
+   "hu-HU": "A karácsonyfa alatt",
+   "pt-BR": "Debaixo da Árvore de Natal",
+   "uk-UA": "Під різдвяною ялинкою",
+   "zh-CN": "圣诞树下"
+  },
+  "park": {
```
Comment (Gymnasiast) [inline objects/official/scenario_text/official.scenario_text.beneath_the_christmas_tree.json:26]:
```suggestion
  "park_name": {
```

File: objects/rct2/scenario_text/rct2.scenario_text.alpine_adventures.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
+{
+ "id": "rct2.scenario_text.alpine_adventures",
+ "authors": [
+  "Chris Sawyer"
+ ],
+ "version": "1.0",
+ "sourceGame": [
+  "rct2"
+ ],
```
Comment (Gymnasiast) [inline objects/rct2/scenario_text/rct2.scenario_text.alpine_adventures.json:9]: For all scenarios that have one, I’d like to have the corresponding `originalId` here.

(The DAT examiner may help:) https://goes.rctspace.com/dat-examiner

File: objects/official/scenario_text/official.scenario_text.beneath_the_christmas_tree.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+ "authors": [
+  "Chris Sawyer"
+ ],
+ "version": "1.0",
+ "sourceGame": [
+  "official"
+ ],
+ "objectType": "scenario_text",
+ "strings": {
+  "name": {
```
[the remark this answers is not in the document]
Comment (AaronVanGeffen) [reply objects/official/scenario_text/official.scenario_text.beneath_the_christmas_tree.json:12]: I wouldn't mind the change, but I was thinking the object itself needs a name as well, right? Thought it best to let the scenario name double as the object name.

**Comment by Gymnasiast** [comment]
Note to self:
- There are 163 added files in this PR
- There are 137 STEX files in RCTC
- A difference of 26
- 18 of these are UCES scenarios, leaving 8
- Tycoon Park and Panda World are present in RCTC, but have no STEX, leaving 6
- Mega Park, Urban Jungle, Fort Anachronism, Alton Tower, Heide-Park and BPB  are not present in RCTC, leaving 0.

File: objects/official/scenario_text/uces.scenario_text.beneath_the_christmas_tree.json [hunk 4]
```diff
@@ -0,0 +1,61 @@
+{
+    "id": "uces.scenario_text.beneath_the_christmas_tree",
+    "authors": [
+        "Chris Sawyer"
```
Comment (Gymnasiast) [inline objects/official/scenario_text/uces.scenario_text.beneath_the_christmas_tree.json:4]: Eh, no :')

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**